packages/preview-middleware/package.json

```diff
@@ -15,7 +15,9 @@
     "main": "dist/index.js",
     "scripts": {
         "start:fixture": "ui5 serve --config test/fixtures/simple-app/ui5.yaml",
-        "build": "tsc --build",
+        "build": "pnpm build:middleware && pnpm build:client",
```

Comment:
above should also be 
`"license": "SEE LICENSE IN LICENSE",`

Reply:
No, it is a middleware like all the others, why would we change the licensing here?